### PR TITLE
Add (way) more settings in the frontend

### PR DESCRIPTION
**Pull request by JensvandeWiel**

### CODE REVIEW

File: frontend/src/pages/server/General.tsx [hunk 1]
```diff
@@ -6,42 +6,162 @@ import {
...
 
 }
 
-function ServerNameAndPasswordsCard({ setServ, serv }: {setServ: React.Dispatch<React.SetStateAction<server.Server>>, serv: server.Server}) {
+function GeneralSettings({ setServ, serv }: {setServ: React.Dispatch<React.SetStateAction<server.Server>>, serv: server.Server}) {
     return (
         <Card variant="soft"  className={''}>
             <Typography level="title-md">
-                Server Name and Passwords
+            Server Name and Passwords
```
Comment (ItsMePepijn) [inline frontend/src/pages/server/General.tsx:28]: indentation

File: frontend/src/pages/server/Multipliers.tsx
Comment (ItsMePepijn) [inline frontend/src/pages/server/Multipliers.tsx:1]: This file seems very recursive

Comment (ItsMePepijn) [reply frontend/src/pages/server/Multipliers.tsx:1]: im sure there is a way to make this more dynamic

Comment (JensvandeWiel) [reply frontend/src/pages/server/Multipliers.tsx:1]: Yes and no, they are sorted per category, there is no obvious/easy way to do this automatically, Its good enough like this for now, as for maybe refactoring it into a slider component it is useless, we need to be able to customize the certain attributes that can change per multiplier

**Pull request merged**